src/py/flwr/client/keras_client.py

```diff
@@ -108,13 +111,17 @@ class KerasClientWrapper(Client):
 
     def __init__(self, keras_client: KerasClient) -> None:
         self.keras_client = keras_client
+        self.properties: Properties = {"tensor_type": "numpy.ndarray"}
 
     def get_parameters(self) -> ParametersRes:
         """Return the current local model parameters."""
         weights = self.keras_client.get_weights()
         parameters = weights_to_parameters(weights)
         return ParametersRes(parameters=parameters)
 
+    def get_properties(self, ins: PropertiesIns) -> PropertiesRes:
+        return PropertiesRes(properties=self.properties)
```

Comment:
`KerasClientWrapper` should ideally remain stateless (apart from the `keras_client` property, see comments on `NumPyClientWrapper` for comparison)